crates/ln-dlc-node/src/transaction.rs

```diff
@@ -4,12 +4,60 @@ use std::fmt::Display;
 use std::fmt::Formatter;
 use time::OffsetDateTime;
 
-#[derive(Debug, Clone, PartialEq)]
+#[derive(Debug, Clone, Copy, PartialEq)]
```

Comment:
Why would we want to `Copy` this type?

Reply:
It means we don't have to `clone` here:

https://github.com/get10101/10101/blob/ee2d5e8c67a0c109145a234fc83b97c5e5928080/crates/ln-dlc-node/src/shadow.rs#L58